src/overlays/actors/ovl_En_Dekubaba/z_en_dekubaba.c

```diff
@@ -121,85 +133,1191 @@ static ColliderJntSphInit D_809E8FC8 = {
+    /* Fire arrow    */ DMG_ENTRY(4, DEKUBABA_DMGEFF_FIRE),
+    /* Ice arrow     */ DMG_ENTRY(2, DEKUBABA_DMGEFF_NONE),
+    /* Light arrow   */ DMG_ENTRY(2, DEKUBABA_DMGEFF_NONE),
+    /* Unk arrow 1   */ DMG_ENTRY(2, DEKUBABA_DMGEFF_NONE),
+    /* Unk arrow 2   */ DMG_ENTRY(2, DEKUBABA_DMGEFF_NONE),
+    /* Unk arrow 3   */ DMG_ENTRY(2, DEKUBABA_DMGEFF_NONE),
+    /* Fire magic    */ DMG_ENTRY(4, DEKUBABA_DMGEFF_FIRE),
+    /* Ice magic     */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NONE),
+    /* Light magic   */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NONE),
+    /* Shield        */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NONE),
+    /* Mirror Ray    */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NONE),
+    /* Kokiri spin   */ DMG_ENTRY(1, DEKUBABA_DMGEFF_SWORD),
+    /* Giant spin    */ DMG_ENTRY(4, DEKUBABA_DMGEFF_SWORD),
+    /* Master spin   */ DMG_ENTRY(2, DEKUBABA_DMGEFF_SWORD),
+    /* Kokiri jump   */ DMG_ENTRY(2, DEKUBABA_DMGEFF_SWORD),
+    /* Giant jump    */ DMG_ENTRY(8, DEKUBABA_DMGEFF_SWORD),
+    /* Master jump   */ DMG_ENTRY(4, DEKUBABA_DMGEFF_SWORD),
+    /* Unknown 1     */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NONE),
+    /* Unblockable   */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NONE),
+    /* Hammer jump   */ DMG_ENTRY(4, DEKUBABA_DMGEFF_NONE),
+    /* Unknown 2     */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NONE),
+};
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_F32(targetArrowOffset, 1500, ICHAIN_STOP),
+};
+
+void EnDekubaba_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnDekubaba* this = THIS;
+    s32 i;
+
+    Actor_ProcessInitChain(&this->actor, sInitChain);
+    ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawCircle, 22.0f);
+    SkelAnime_Init(globalCtx, &this->skelAnime, &gDekuBabaSkel, &gDekuBabaFastChompAnim, this->jointTable,
+                   this->morphTable, 8);
+    Collider_InitJntSph(globalCtx, &this->collider);
+    Collider_SetJntSph(globalCtx, &this->collider, &this->actor, &sJntSphInit, this->colliderElements);
+
+    if (this->actor.params == DEKUBABA_BIG) {
+        this->size = 2.5f;
+
+        for (i = 0; i < sJntSphInit.count; i++) {
+            this->collider.elements[i].dim.worldSphere.radius = this->collider.elements[i].dim.modelSphere.radius =
+                (sJntSphElementsInit[i].dim.modelSphere.radius * 2.50f);
+        }
+
+        // This and its counterpart below mean that a Deku Stick jumpslash will not trigger the Deku Stick drop route.
+        // (Of course they reckoned without each age being able to use the other's items, so Stick and Master Sword
+        // jumpslash can give the Stick drop as adult, and neither will as child.)
+        if (LINK_IS_CHILD) {
+            sBigDekuBabaDamageTable.table[0x1B] = DMG_ENTRY(4, DEKUBABA_DMGEFF_NONE); // DMG_JUMP_MASTER
+        }
+
+        CollisionCheck_SetInfo(&this->actor.colChkInfo, &sBigDekuBabaDamageTable, &sColChkInfoInit);
+        this->actor.colChkInfo.health = 4;
+        this->actor.naviEnemyId = 8; // Big Deku Baba
```

Comment:
```suggestion
        this->actor.naviEnemyId = 0x08; // Big Deku Baba
```
Per #775
> All naviEnemyIds are now hex

also below